techniques/general/device-motion-sensor-input.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Provide conventional controls and an application setting for motion activated input</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Provide conventional controls and an application setting for motion activated input</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id"></p>
+    </section>
+		<section id="applicability">
+			<h2>When to Use</h2>
+			<p>Use this technique on web pages that detect device or user motion such as shaking or tilting and use this motion as a means of input.  If the motion itself is essential to the application's function, then this technique does not apply.</p>
+			<p>This technique also does not relate to movement of users through space as registered by geolocation sensors or beacons, or events observed by the device other than intentional gesturing by the user. It also does not cover indirect motion associated with operating a keyboard, pointer, or assistive technology.</p>
+		</section>
+
+		<section id="description">
+
+			<h2>Description</h2>
+	
+			<p>The objective of this technique is to ensure that:</p>
+			<ol>
+				<li>People who use mounted devices or who cannot precisely manipulate a device are able to provide inputs through more conventional user interface components, and</li>
+				<li>People who may accidentally activate sensors due to tremors or other motor impairments have the ability to turn off motion actuation to prevent such accidental triggering of functions.</li>
+			</ol>
+			
+			<p>In this technique, when a device sensor such as accelerometers or gyroscope is used to gather input:</p>

```

Comment:
reads awkwardly. perhaps better "when device sensors such as accelerometers or gyroscopes are used"

Reply:
alternatively: "when a device sensor such as an accelerometer or gyroscope is used"